src/openzaak/components/besluiten/tests/test_auth.py

```diff
@@ -263,6 +267,9 @@ class ExternalBesluittypeScopeTests(JWTAuthMixin, APITestCase):
     component = ComponentTypes.brc
 
     def test_besluit_list(self):
+        Service.objects.create(
+            api_type=APITypes.ztc, api_root="https://externe.catalogus.nl/api/v1/"
```

Comment:
you could use the constant here too - and maybe move this to `setUpTestData` instead?